six/modules/c++/six.sidd/source/GeographicAndTarget.cpp

```diff
@@ -45,10 +45,7 @@ bool GeographicCoverage::operator==(const GeographicCoverage& rhs) const
 bool GeographicAndTarget::operator==(const GeographicAndTarget& rhs) const
 {
     return (geographicCoverage == rhs.geographicCoverage &&
-        targetInformation == rhs.targetInformation && 
-        earthModel == rhs.earthModel &&
-        imageCorners == rhs.imageCorners &&
-        validData == rhs.validData &&
+        targetInformation == rhs.targetInformation &&
```

Comment:
why?

Reply:
All of those were specific to SIDD 2.0.0 only, and are all in GeoDataBase now.
We were using the same class for both versions before because of the schema being weird ...